packages/server/src/lib/addTenantScript.js

```diff
@@ -0,0 +1,39 @@
+require('dotenv').config();
+
+const db = require('../db');
+
+const userInputArgs = process.argv.slice(2);
+
+const addNewTenant = async (args) => {
+    const nameArg = args[0];
+    const agencyIdArg = args[1];
+    try {
+        const isValidTenantName = nameArg && nameArg.match(/tenantName=[\w\d\s]+/);
+        const isValidAgency = agencyIdArg && agencyIdArg.match(/agencyId=\d+/);
+        if (!isValidTenantName || !isValidAgency) {
+            throw new Error('Valid tenant name and agency ID is required');
+        }
+        const tenantName = nameArg.slice(11);
+        const tenantAgencyId = Number(agencyIdArg.slice(9));
+
+        const agency = await db.getAgency(tenantAgencyId);
+
+        if (!agency.length) {
+            throw new Error('No Agency with this ID exists');
+        }
+        const tenant = {
+            display_name: tenantName,
+            main_agency_id: tenantAgencyId,
+        };
+        const result = await db.createTenant(tenant);
```

Comment:
im not familiar with tenant terminology maybe tomorrow yall can ramp me up on this. I thought tenants were top level entities (Tenant > Agencies) so in this case how does tenantAgencyId exists?

Reply:
A _tenant_ can be thought of as an isolated namespace/world for a given state using GOST. Within one, users can only see and manage agencies etc. within the same tenant -- the only cross-tenant actions that exist are creation of new tenants, which only USDR super-admins will be allowed to do.

As such, I updated this script to create a new root agency in the new tenant, as opposed to moving an existing agency from one tenant to another as you commented on.